src/DataViews/LiveFunctionsDataView.cpp

```diff
@@ -116,9 +117,9 @@ std::string LiveFunctionsDataView::GetValue(int row, int column) {
     case kColumnStdDev:
       return orbit_display_formats::GetDisplayTime(absl::Nanoseconds(stats.ComputeStdDevNs()));
     case kColumnModule:
-      return function.module_path();
+      return function == nullptr ? "" : function->module_path();
     case kColumnAddress:
-      return absl::StrFormat("%#x", function.address());
+      return function == nullptr ? "" : absl::StrFormat("%#x", function->address());
```

Comment:
We collect a PC from manual instrumentation as well. Should we use it here at some point (if it's unique after aggregation)?

Reply:
Same for the module (again, if it is unique).

Reply:
I guess, that's for a subsequent PR, right?

Reply:
> I guess, that's for a subsequent PR, right?

Yes yes (hence my "at some point").